common/Server/eos/core/eos_core.sqf

```diff
@@ -74,7 +78,11 @@ if (_debug) then {diag_log (format ["Spawned House Patrol: %1",_counter]);0= [_m
 							if (_bMin > 0) then {
 								  _pos=[_mkr,50] call EOS_fnc_findSafePos;
 									_bGroup=[_pos,_bSize,_faction,_side,true] call EOS_fnc_spawngroup;
-										0 = [_bGroup,_mkr] call EOS_fnc_taskpatrol;
+										if(isClass(configfile >> "lambs_main_Display")) then {
```

Comment:
We should probably do the same format as detecting ace though the mod path config just encase lambs changes